elements/rh-alert/demo/rh-alert.html

```diff
@@ -5,7 +5,7 @@
   <h2 slot="header">Alert</h2>
 
   <rh-alert state="default">
-    <h3 slot="header">Default</h3>
+   <h3 slot="header">Default</h3>
     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est
       egestas, a sollicitudin mauris tincidunt.</p>
     <button slot="actions" data-action="dismiss">Dismiss</button>
```

Comment:
reserving this comment to block merge pending @brianferry 's review with design re: header actions vs footer actions. resolve to automerge

Reply:
Resolving this, talked to Marionne and we agreed to keep both the header actions (close button) and the footer actions.